test/e2e/autotls/auto_tls_test.go

```diff
@@ -91,6 +93,41 @@ func testAutoTLS(t *testing.T) {
 	rootCAs := createRootCAs(t, clients, objects.Route.Namespace, certName)
 	httpsClient := createHTTPSClient(t, clients, objects, rootCAs)
 	testingress.RuntimeRequest(t, httpsClient, "https://"+objects.Service.Status.URL.Host)
```

Comment:
Could you help to update the URL to `objects.Service.Status.URL.String()` here?

Reply:
Will do.